src/TTrigger.cpp

```diff
@@ -445,7 +445,7 @@ bool TTrigger::match_begin_of_line_substring(const QString& toMatch, const QStri
     if (toMatch.startsWith(regex)) {
         std::list<std::string> captureList;
         std::list<int> posList;
-        captureList.push_back(regex.toLatin1().data());
+        captureList.emplace_back(regex.toLatin1().data());
```

Comment:
Better mark that line - what on earth is `toLatin()` doing there, and it can be a `constData()` also I suppose...! Guess that is something to fix for I18n!